tests/openvino/test_exporters_cli.py

```diff
@@ -131,6 +131,14 @@ class OVCLIExportTestCase(unittest.TestCase):
             (13,),
             (16,),
         ),
+        (
+            "text-generation",
+            "llama",
+            "int8",
+            "--dataset wikitext2 --trust-remote-code",
+            (13,),
+            (16,),
+        ),
```

Comment:
Could you remove `--num-samples 1` from the previous test instead of adding this one? This will have less impact on testing time and maintain the same coverage.

Reply:
Done.